src/main/java/com/github/tomakehurst/wiremock/common/xml/Xml.java

```diff
@@ -99,17 +99,9 @@ public static String prettyPrint(Document doc) {
   }
 
   private static TransformerFactory createTransformerFactory() {
-    try {
-      TransformerFactory transformerFactory =
-          (TransformerFactory)
-              Class.forName("com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl")
-                  .getDeclaredConstructor()
-                  .newInstance();
-      transformerFactory.setAttribute("indent-number", 2);
-      return transformerFactory;
-    } catch (Exception e) {
-      return TransformerFactory.newInstance();
```

Comment:
PlainTextDiffRendererTest.wrapsLargeXmlBodiesAppropriatelyJre11() would fail with the following exception without this change:
```
java.lang.IllegalAccessException: class com.github.tomakehurst.wiremock.common.xml.Xml cannot access class com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl (in module java.xml) because module java.xml does not export com.sun.org.apache.xalan.internal.xsltc.trax to unnamed module @37858383
```